### PR TITLE
added input reader, renamed twoDArray to TwoDArray and wrote main

### PR DESCRIPTION
**Pull request by jkomendant**

### CODE REVIEW

**Review by shweinfeld — COMMENTED** [review]
Why are you saving it in array and then traversing to add to the 2D Array?  Cant you add the integers to the array directly?

**Comment by jkomendant** [comment]
> Why are you saving it in array and then traversing to add to the 2D Array? Cant you add the integers to the array directly?

When I did it initially like that, all the old numbers got overwritten by the newest number. By the time the array was complete, it was entirely filled with the number 45

**Review by shweinfeld — APPROVED** [review]

File: src/max/sum/TwoDArray.java [hunk 1]
```diff
@@ -1,21 +1,22 @@
 package max.sum;
 
-public class twoDArray {
+public class TwoDArray {
     int[][] rectangle;
     int size;
```
Comment (lliebman) [inline src/max/sum/TwoDArray.java:5]: make rectangle and size private?

Comment (jkomendant) [reply src/max/sum/TwoDArray.java:5]: A good point. @shweinfeld was the author of this code

**Pull request merged**